### PR TITLE
Remove useless return

### PR DESCRIPTION
**Pull request by MathiasReker**

### CODE REVIEW

File: classes/Tools14.php [hunk 1]
```diff
@@ -2093,15 +2093,15 @@ public static function jsonEncode($data)
      */
     public static function displayAsDeprecated()
     {
-        return;
+
```
Comment (PierreRambaud) [inline classes/Tools14.php]: Useless spaces.

File: classes/Tools14.php [hunk 2]
```diff
@@ -2093,15 +2093,15 @@ public static function jsonEncode($data)
...
+        
     }
 
     /**
      * Display a warning message indicating that the parameter is deprecated.
      */
     public static function displayParameterAsDeprecated($parameter)
     {
-        return;
+
```
Comment (PierreRambaud) [inline classes/Tools14.php]: Useless spaces.

File: classes/pclzip.lib.php [hunk 3]
```diff
@@ -226,7 +226,7 @@ public function PclZip($p_zipname)
           $this->magic_quotes_status = -1;
 
           // ----- Return
-          return;
+
```
Comment (PierreRambaud) [inline classes/pclzip.lib.php]: Useless spaces.

Comment (MathiasReker) [reply classes/pclzip.lib.php]: Thanks. Fixed in my script.

**Comment by MathiasReker** [comment]
thank you pie

**Comment by MathiasReker** [comment]
Fixed in script. New PR: https://github.com/PrestaShop/autoupgrade/pull/267

**Pull request closed without merging**